docs/develop/sdk/1-dev-setup.md

```diff
@@ -45,7 +44,7 @@ Copy the above code into index.ts.
 You can know execute the script:
 
 ```bash
-ts-node index.ts
+yarn exec ts-node index.ts
```

Comment:
I think simply `yarn ts-node index.ts` works, no?

Reply:
Might be? I always do exec because it's not a script in the `package.json` and `yarn xyz` most of the time refer to something defined in `scripts` section

Reply:
That's what we also explain everywhere in the workshop, e.g. in https://github.com/KILTprotocol/docs/blob/develop/docs/develop/workshop/04-claimer/02-request.md?plain=1#L60=

Reply:
[efd821e](https://github.com/KILTprotocol/docs/pull/81/commits/efd821e68190b96f07c04eefcbec234ec4bb68c2)